techniques/general/G220-display-values.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Displaying the previously entered value</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+	</head>
+	<body>
+		<h1>Displaying the previously entered value</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p class="instructions">Provide information below to help editors associate the technique properly. Contents of the meta section are not output by the processor.</p>
+			<p id="sc">3.3.8 Redundant entry</p>
+			<p id="type">Sufficient</p>
+		</section>
+		<section id="applicability">
+			<h2>Applicability</h2>
+			<p class="instructions">Any technology that supports scripting.</p>
+		</section>
+		<section id="description">
+			<h2>Description</h2>
+			<p>The objective of this technique is to display information that was previously provided by the user or by the system rather than requiring the user to remember the information from a previous step.  This improves success for individuals with cognitive and learning disabilities and memory impairments.</p>
+
+			<p>This technique best applies when information is almost always the same as information previously entered or provided.</p>
```

Comment:
This is confusing. Are we requiring this when information isn't the same? Seems like this technique _only_ applies when the information is exactly the same, right?

Reply:
Removed that line, doesn't seem to help anything.